gateway/host_checker.go

```diff
@@ -136,8 +136,8 @@ func (h *HostUptimeChecker) execCheck() {
 	h.resetListMu.Unlock()
 	for _, host := range h.HostList {
 		_, err := h.pool.SendWork(host)
-		if err != nil {
-			log.Errorf("[HOST CHECKER] could not send work, error: %v", err)
+		if err != nil && err != tunny.ErrPoolNotRunning {
+			log.Warn("[HOST CHECKER] could not send work, error: %v", err)
```

Comment:
This error in check is totally valid. It should be Warnf function.

Reply:
@buger why should we Warn the msg if the pool is not running?